causalpy/tests/test_integration_pymc_examples.py

```diff
@@ -27,6 +36,18 @@ def test_did():
 
 @pytest.mark.integration
 def test_did_banks_simple():
+    """
+    Test simple Differences In Differences Experiment on the 'banks' data set.
+
+    formula="bib ~ 1 + district * post_treatment"
```

Comment:
I think we can get this line formatted as code (i.e. monoface font) using a sphinx role or directive